src/views/UserSettings.vue

```diff
@@ -9,6 +9,15 @@
         <v-card :title="$t('UserSettings.ChangeLanguage')" prepend-icon="mdi-earth">
           <v-divider></v-divider>
           <v-card-text>
+            <v-autocomplete
+              v-model="userSettingsForm.selectedCountry"
+              label="Country"
```

Comment:
translation missing :)

Reply:
Thanks for seeing this. Corrected. Translation key is already in locale files (from a previous PR)